ZacGormanrewrite.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<html>
+	<body>
+		<h1>
```

Comment:
An h1 should not wrap a section or p tag. An h1 should most over time only contain text.